CONTRIBUTING.md

```diff
@@ -11,14 +12,22 @@ If you want to learn more about this project, read the [wiki](https://github.com
 
 ## Contributions
 
+Please review this [diagram](https://drive.google.com/file/d/1RjCeUwJ0s3huq1iRSXwdy02wD54HKNxA/view?usp=sharing), to better understand the git workflow.
+
 We would be pleased to accept code contributions to this project.
-As we are in the alpha stage, the main priority is getting a robust blockchain up and running, with as many of the mainnet features as possible.
+As we have now released, the main priority is improving the mainnet blockchain.
 You can visit our [Trello project board](https://trello.com/b/ZuNx7sET) to get a sense of what is in the backlog.
 Generally things to the left are in progress or done. Some things go through "Coming up soon" but some will come directly out of other columns.
 Usually the things closer to the top of each column are the ones that will be worked on soonest.
 If you are interested in cryptography, math, or just like hacking in python, there are many interesting problems to work on.
 Contact any of the team members on [Keybase](https://keybase.io/team/chia_network.public), which we use as the main communication method and you can comment on any Trello card.
 
+We ask that external contributors create a fork of the `main` branch for any feature work they wish to take on.
+
+Members of the Chia organization may create feature branches from the `main` branch.
```

Comment:
As long as Chia organization members can create new branches, we need a strategy for branch garbage collection. We currently have > 30 branches, many of which are likely stale and can be cleaned up. It's better if people are required to clean up their own messes, or maybe automatically notified when they fail to do so. Not sure what the best solution is though.

Reply:
PEP440 requires pre release of the 1.1 version to use something like b or rc. It should be 1.1b1 to start though.

Reply:
So the "main" branch replaces what the dev branch was before? I.e. the main branch can potentially contain code that has not been thoroughly tested.

Reply:
@mariano54 That's my understanding, and it's a common workflow, maybe even the standard github workflow.

Reply:
I am currently cleaning this up by hand. but will write automation.

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/1376#discussion_r598466485

@mariano54 the main branch will have branch protection just like dev does now. this is the place where the long term testing will take place with married changes via a beta testnet. 

the release branches v1.0.x et al will contain the code you clone if you wish to run a stable version of chia. No one not writing code should be nor will be instructed to pull main. We will also move away from git clone for installs, to installers for all operating systems, within the week.

Reply:
> 
> 
> I like the fact that the main branch is considered "stable". This is different than what we have been doing with the dev branch. We need to thoroughly review and test PRs before merging. I think what we were doing before was the only real way to get done with the features that were necessary. But now since we have a real mainnet and funds to secure, we have to increase our testing requirements.

I agree whole heartedly. its why i am introducing `beta  testnet` review environments. for both main, and for all feature branches. The reviewer simply needs to ask to see this change running live on a review environment. and we can let suggested changes run as long as needed in their own eco system with peers we can run various tests with. we can be reasonably certain that these changes are not destructive. The second layer of running a long running beta net on the `main` layer gives us the ability to see all future code being proposed as a release candidate married to the code base and running in the same patterns. Its the best possible way IMO to ensure were not introducing faults into the production ecosystem.